File: cpp/35c_1703B_ICPCBalloons.cpp
#include<bits/stdc++.h>

int main(){
    int t, n, balloons;
    std::string s;
    std::vector<int> ans;
    std::cin>>t;
    for(int i = 0; i < t; i++){
        std::cin>>n;
        std::cin>>s;
        std::unordered_set<char> hset;
        balloons = n;
        for (auto j : s){
            if (hset.find(j) == hset.end()){
                balloons ++;
                hset.insert(j);
            }
        }
        ans.push_back(balloons);
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/35c_1703B_ICPCBalloons.py
def main():
    ans = []
    t = int(input())
    for _ in range(t):
        n = int(input())
        s = input().strip()
        balloons = n
        hset = set()
        for i in s:
            if i not in hset:
                balloons += 1
                hset.add(i)
        ans.append(balloons)
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()